src/microbe_stage/ProcessSpeedInformation.cs

```diff
@@ -15,8 +15,8 @@ public ProcessSpeedInformation(BioProcess process)
 
     public string Name => Process.Name;
 
-    public Dictionary<Compound, float> WritableInputs { get; } = new();
-    public Dictionary<Compound, float> WritableOutputs { get; } = new();
+    public Dictionary<Compound, float> WritableInputs { get; set; } = new();
+    public Dictionary<Compound, float> WritableOutputs { get; set; } = new();
```

Comment:
I'm against this change as this is meant to be a simple data container, it makes this much more complicated to use code if the properties can be reassigned by any piece of code in the entire game.

Reply:
What would you suggest? I don't want to make any complicated changes really...

Reply:
Remove these setters. Then deal with any code that doesn't work anymore in a different way at the code site that breaks.